setup.cfg

```
@@ -16,13 +16,15 @@ classifiers =
 
 [options]
 packages = find:
+py_modules = xpartition
 zip_safe = False  # https://mypy.readthedocs.io/en/latest/installed_packages.html
 include_package_data = True
 python_requires = >=3.6
```

If you don't mind updating this too -- I think xarray requires at least Python 3.7 these days (plus this PR uses dataclasses):
```suggestion
python_requires = >=3.7
```

---

xarray 0.16.2 supports python 3.6, and we still use it in the prognostic run image. Is there a way to install `dataclasses` only for python 3.6?